demos/sketch-rnn-js/README.md

```diff
@@ -0,0 +1,366 @@
+|backpack|barn|basket|bear|bee|
+|beeflower|bicycle|bird|book|brain|
+|bridge|bulldozer|bus|butterfly|cactus|
+|calendar|castle|cat|catbus|catpig|
+|chair|couch|crab|crabchair|crabrabbitfacepig|
+|cruise_ship|diving_board|dog|dogbunny|dolphin|
+|duck|elephant|elephantpig|eye|face|
+|fan|fire_hydrant|firetruck|flamingo|flower|
+|floweryoga|frog|frogsofa|garden|hand|
+|hedgeberry|hedgehog|helicopter|kangaroo|key|
+|lantern|lighthouse|lion|lionsheep|lobster|
+|map|mermaid|monapassport|monkey|mosquito|
+|octopus|owl|paintbrush|palm_tree|parrot|
+|passport|peas|penguin|pig|pigsheep|
+|pineapple|pool|postcard|power_outlet|rabbit|
+|rabbitturtle|radio|radioface|rain|rhinoceros|
+|rifle|roller_coaster|sandwich|scorpion|sea_turtle|
+|sheep|skull|snail|snowflake|speedboat|
+|spider|squirrel|steak|stove|strawberry|
+|swan|swing_set|the_mona_lisa|tiger|toothbrush|
+|toothpaste|tractor|trombone|truck|whale|
+|windmill|yoga|yogabicycle|everything (※)||
+
+※ *gen* only
+
+You should load and parse the json files using whatever method you are comfortable with, and pass the parsed blob into the constructor of the sketch\_rnn object. We will get to this later.
+
+We found it sometimes simpler to copy the contents of the json file and place it into an inline .js code so that the demo loads the model synchronously. Some of our examples below will do this.
+
+## Run Pre-built Examples
+
+There are a number of proof-of-concept demos built to use the Sketch RNN model.  You can look at the corresponding code to study in detail how the model works.  To run these examples, it is recommended to use a simple local webserver, such as the http-server can be obtained using npm, and load the local html file from the local server.  Some examples require this, since they need to dynamically load .json model files, and local static session doesn't allow for this in many browsers.
+
+If you use the http-server, running something would be like putting `http://127.0.0.1:8080/basic_predict.html` in the address tab in Chrome.  For debugging, it is recommended you open a console tab on the side of the screen to look at the log messages.
+
+### 1) basic\_vae.html / basic\_vae.js
+
+This basic demo will generate an unconditional images on the web page given random latent vectors.  In addition, we demonstrate what the an image looks like if we average the two latent vectors.
+
+### 2) basic\_predict.html / basic\_predict.js
+
+Similar to basic\_grid, this demo will keep on generating random vector images unconditionally.  Unlike basic\_vae, each point is generated per time frame (at 30 or 60 fps), while basic\_vae generates the entire image at once.  In basic, you can adjust the "temperature" variable, which controls the uncertainty of the strokes.
+
+### 3) simple_predict.html / simple_predict.js
+
+This demo is also an unconditional generative, that tries to finish the drawing that the user starts.  If the user doesn't draw anything, the computer will keep on drawing stuff from scratch.
```

Comment:
This demo is also generates unconditionally, attempting to finish the drawing that the user starts.